src/Page/Form.purs

```diff
@@ -19,12 +24,14 @@ import Ocelot.Block.Toggle (toggle) as Toggle
 import Ocelot.HTML.Properties (css)
 
 type State =
-  { form :: Either String (Page Expr)
+  { form :: Either String { expr :: Page Expr, evaled :: Page ExprType }
```

Comment:
I'm not sold on the naming here. Happy to change it to whatever if we've got better names.

Reply:
This is better than anything I came up with when I was trying lol